### PR TITLE
create server subpackage

### PR DESCRIPTION
**Pull request by samjxn**
### Summary of changes
Restructured the server directory.
 - created a new directory named `server`
 - moved server.py and models directory to the server directory
 - updated paths in server.py

### CODE REVIEW

File: app/server/server.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 class MainHandler(tornado.web.RequestHandler):
     def get(self):
         loader = tornado.template.Loader(".")
-        self.write(loader.load("client/index.html").generate())
+        self.write(loader.load("./../client/index.html").generate())
```
Comment (jonnykry) [inline app/server/server.py]: bummer, this wasn't working for me because I was missing the `./`

nice

**Comment by jonnykry** [comment]
👍 LGTM, not sure why Travis is failing though.  "bad decrypt"

@gtharris is that expected?

**Comment by samjxn** [comment]
@jonnykry It may be because I moved things around in a way that the travis/GAE config files weren't expecting.  @gtharris can you confirm?

File: app/server/server.py (hunk 1 above)
Comment (samjxn) [reply app/server/server.py]: Yeah, I hit that too lol.  I think this approach was my third attempt

**Comment by gtharris** [comment]
moving server.py should have no effect on the decryption of the keys
I have commited a new key for the gae so this is where our error is likely coming from.
I will check my branch then may push up my changes sooner so that your builds start working again

**Pull request merged**